geode-core/src/integrationTest/java/org/apache/geode/internal/jta/functional/CacheJUnitTest.java

```diff
@@ -577,7 +582,8 @@ public void testScenario7() throws Exception {
       assertEquals("get value mismatch with put", "\"value1\"", tok);
 
       current_fullpath = jtaObj.currRegion.getFullPath();
-      assertEquals("failed retrieving current region fullpath", "/" + DEFAULT_RGN + "/region1",
+      assertEquals("failed retrieving current region fullpath",
+          SEPARATOR + DEFAULT_RGN + "/region1",
```

Comment:
Did you miss this one?, shouldn't it be `SEPARATOR + DEFAULT_RGN + SEPARATOR + "region1"`?